src/main/java/github/weichware10/toolbox/gui/TutorialController.java

```diff
@@ -0,0 +1,85 @@
+package github.weichware10.toolbox.gui;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.layout.HBox;
+import javafx.scene.image.ImageView;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'javafx.scene.image.ImageView' import. Should be before 'javafx.scene.layout.HBox'.